spec/spec_helper.rb

```diff
@@ -3,6 +3,9 @@
 require 'yarjuf'
 
 RSpec.configure do |config|
+  config.filter_run focus: true
+  config.run_all_when_everything_filtered = true
```

Comment:
Slightly strange language, but means: "when no `focus: true` filters are found, everything is filtered, which means, nothing runs, which means _run everything_"